egs/wsj/s5/steps/online/nnet2/train_ivector_extractor.sh

```diff
@@ -41,6 +41,7 @@ num_processes=4 # each job runs this many processes, each with --num-threads thr
 cmd="run.pl"
 stage=-4
 ivector_dim=100 # dimension of the extracted i-vector
+online_cmvn=false # apply online-cmvn on i-vector input features, uses the configuration from UBM,
```

Comment:
.. I think here, though, calling the variable online_cmvn_iextractor might be clearer